services/labels/labels.go

```diff
@@ -201,7 +201,7 @@ func (self *Labeler) SetClientLabel(
 	self.mu.Lock()
 	defer self.mu.Unlock()
 
-	checked_label := strings.ToLower(new_label)
+	checked_label := strings.ToLower(strings.TrimSpace(new_label))
```

Comment:
you probably also need to update the new label which will be inserted below:
```
new_label = strings.TrimSpace(new_label)
```

Reply:
Good Morning, I want some guidance/clarification on this. Are you talking about directly updating the `new_label` for when its used below in the following section?
```
    if !utils.InString(cached.lower_labels, checked_label) {
        cached.record.Label = append(cached.record.Label, new_label)
        cached.lower_labels = append(cached.lower_labels, checked_label)
    }
```

Reply:
Should it also be updated in the `RemoveClientLabel` ? I assumed not since that isn't called by user input but the gui button which has the field prepopulated when called, but what do you think?